index.html

```diff
@@ -0,0 +1,382 @@
+        <span>Subscriptions</span>
+      </div>
+
+      <div class="line"></div>
+
+      <div class="nav-box">
+        <i class="fa-regular fa-file-video"></i>
+        <span>Library</span>
+      </div>
+      <div class="nav-box">
+        <i class="fa-solid fa-clock-rotate-left"></i>
+        <span>History</span>
+      </div>
+      <div class="nav-box">
+        <i class="fa-solid fa-clapperboard"></i>
+        <span>Your videos</span>
+      </div>
+      <div class="nav-box">
+        <i class="fa-regular fa-clock"></i>
+        <span>Watch later</span>
+      </div>
+      <div class="nav-box">
+        <i class="fa-solid fa-chevron-down"></i>
+        <span>Show more</span>
+      </div>
+
+      <div class="line"></div>
+
+      <h1>Explore</h1>
+      <div class="nav-box">
+        <i class="fa-solid fa-fire"></i>
+        <span>Trending</span>
+      </div>
+      <div class="nav-box">
+        <i class="fa-solid fa-music"></i>
+        <span>Music</span>
+      </div>
+      <div class="nav-box">
+        <i class="fa-solid fa-gamepad"></i>
+        <span>Gaming</span>
+      </div>
+      <div class="nav-box">
+        <i class="fa-solid fa-trophy"></i>
+        <span>Sports</span>
+      </div>
+      <div class="nav-box">
+        <i class="fa-solid fa-bullhorn"></i>
+        <span>Live</span>
+      </div>
+      <div class="nav-box">
+        <i class="fa-regular fa-newspaper"></i>
+        <span>News</span>
+      </div>
+      <div class="nav-box">
+        <i class="fa-regular fa-lightbulb"></i>
+        <span>Learning</span>
+      </div>
+      <div class="nav-box">
+        <i class="fa-solid fa-shirt"></i>
+        <span>Fashion & Beauty</span>
+      </div>
+
+      <div class="line"></div>
+
+      <div class="nav-box">
+        <i class="fa-solid fa-gear"></i>
+        <span>Setting</span>
+      </div>
+      <div class="nav-box">
+        <i class="fa-regular fa-font-awesome"></i>
+        <span>Report history</span>
+      </div>
+      <div class="nav-box">
+        <i class="fa-regular fa-circle-question"></i>
+        <span>Help</span>
+      </div>
+      <div class="nav-box">
+        <i class="fa-solid fa-circle-info"></i>
+        <span>Send feedback</span>
+      </div>
+      <div class="line"></div>
+
+      <p>About Press Copyright Contact us Creators Advertise Developers</p>
+      <p>Terms Privacy Policy & Safety How YouTube works Test new features</p>
+      <p>@ 2022 Google LLC</p>
+
+    </nav>
+
+    <main class="container">
+      <!-- catagory -->
+      <div id="catagory">
+        <span>All</span>
+        <span>New to you</span>
+        <span>Mixes</span>
+        <span>Music</span>
+        <span>Lana Del Rey</span>
+        <span>Indie Music</span>
+        <span>The Weeknd</span>
+        <span>Rhythm & Blues</span>
+        <span>Live</span>
+        <span>4K resolution</span>
+        <span>Chill-out Music</span>
+        <span>Background Music</span>
```

Comment:
Instead of using plain <span> tag, I think your span should have a link (<a> tag). This is for three reasons: convention, semantics, and user experience(UX). 
I would suggest you refactor your code(line 144 to 155) to look something like this 👇🏽

<span><a href="#">All</a></span>